cslbot/helpers/textutils.py

```diff
@@ -368,12 +368,34 @@ def gen_sanitize(msg):
 def gen_intensify(msg):
     return "[%s INTENSIFIES]" % msg.upper()
 
+def gen_djones(msg):
+    keyboard = [
+            'qwertyuiop',
+            '\0asdfghjkl',
+            '\0\0zxcvbnm\0\0',
+            ]
+    outmsg = ''
+    for letter in msg:
+        if random() < 0.15 and letter in ''.join(keyboard):
+            r = (0 if letter in keyboard[0] else (1 if letter in keyboard[1] else 2))
```

Comment:
These variable names are a bit short, this isn't go :P

Reply:
just nr, nc? or others?

Reply:
all the 1/2 char variables, honestly